frontend/src/service/apiService.ts

```diff
@@ -0,0 +1,27 @@
+export const get = async (url: String, token: String | undefined) => {
+  const response = await fetch(url as RequestInfo, {
+    headers: { Authorization: `Bearer ${token}` },
+  });
+  return response;
+};
+
+export const put = async (url: String, token: String | undefined) => {
+  const response = await fetch(url as RequestInfo, {
+    method: 'PUT',
+    headers: { Authorization: `Bearer ${token}` },
+  });
+  return response;
+};
+
+export const post = async (
+  url: String,
+  token: String | undefined,
```

Comment:
The syntactic sugar for optional arguments which you can try is
`token?: string`
Also, I thought `string` was lower-case in typescript. Is upper-case OK as well?

Reply:
After a bit of reading, you shouldn't use capitals as it refers to Object types rather than the primitive types or something like that. I'll change all of these.

I think JS is just being nice.